mono/metadata/mono-perfcounters.c

```diff
@@ -1378,7 +1383,9 @@ mono_perfcounter_get_sample (void *impl, MonoBoolean only_value, MonoCounterSamp
 }
 
 gint64
-mono_perfcounter_update_value (void *impl, MonoBoolean do_incr, gint64 value)
+ves_icall_System_Diagnostics_PerformanceCounter_UpdateValue (
+	void *impl, MonoBoolean do_incr, gint64 value, MonoError *error)
+// previously mono_perfcounter_update_value
```

Comment:
Why keeping that?

Reply:
It is much easier to sere here than in history and it has been useful looking at other branches at the same time.